packages/figma-block/src/FigmaEmptyBlock.tsx

```diff
@@ -12,13 +12,15 @@ export const FigmaEmptyBlock = ({ onOpenAssetChooser }: Props): ReactElement =>
     return (
         <button
             data-test-id="figma-empty-block"
-            className="tw-group tw-w-full tw-py-16 tw-px-4 tw-border-dashed tw-border tw-cursor-pointer tw-text-center tw-border-line-x-strong hover:tw-border-black"
+            className="tw-group tw-w-full tw-py-16 tw-px-4 tw-border-dashed tw-border tw-cursor-pointer tw-text-center tw-border-line-x-strong hover:tw-border-[color-mix(in_srgb,_var(--f-theme-settings-body-color)_70%,_transparent)]"
             onClick={onOpenAssetChooser}
         >
-            <div className="tw-text-xl tw-mb-4 tw-flex tw-justify-center tw-text-black-40 group-hover:tw-text-violet-60">
+            <div className="tw-text-xl tw-mb-4 tw-flex tw-justify-center tw-text-[var(--f-theme-settings-body-color)] group-hover:tw-text-[color-mix(in_srgb,_var(--f-theme-settings-body-color)_70%,_transparent)]">
```

Comment:
You got it the other way around.

Now by default the icon is 100% opacity, and when hovering it gets to 70%.